lib/bddsm/describe.rb

```diff
@@ -14,5 +17,11 @@ def run
     def it(&)
       Execution.new(describe: self, &).run
     end
+
+    def let(name, &)
+      self.singleton_class.instance_eval { attr_accessor name.to_sym }
+      @memoized_lets[name] ||= instance_exec(&)
```

Comment:
Looks like we have double execution in here. Please add a test to check if let block execute only once

Reply:
Do you mean  lib/bddsm/describe.rb L23,24
 did [a6f671c] fixed it?

Reply:
Highly likely it has fixed it. lease add a test to check if let block executed only once